deploy/cloudformation/README.md

```diff
@@ -19,7 +19,6 @@ FLEET_URL="<Elastic Agent Fleet URL>"
 ENROLLMENT_TOKEN="<Elastic Agent Enrollment Token>"
 ELASTIC_ARTIFACT_SERVER="https://artifacts.elastic.co/downloads/beats/elastic-agent" # Replace artifact URL with a pre-release version (BC or snapshot)
 ELASTIC_AGENT_VERSION="<Elastic Agent Version>" # e.g: 8.8.0 | 8.8.0-SNAPSHOT
-# INTEGRATION=CloudSecurityPostureManagement # Defaults to VulnerabilityManagement if not specified
```

Comment:
I think you accidentally reverted this file to an older version?

Reply:
I did revert the file, but I also removed this option so it isn't relevant anymore.
Since we have multiple templates this will be configured using the `DeploymentType` and because it isn't a template parameter, I did not write or document it here.